finetuner-workflow/finetuner/finetuner.py

```diff
@@ -988,7 +999,7 @@ def evaluate(
     gradient_checkpointing=True,
 
     # Learning rate scheduler arguments
-    warmup_steps=8,
+    warmup_steps=warmup_steps,
```

Comment:
`TrainingArguments` supports specifying the `warmup_ratio` directly as a float:
```suggestion
    warmup_ratio=float(args.warmup_ratio),
```